### PR TITLE
Minor schema changes

### PR DESCRIPTION
**Pull request by Bahugunajii**
### Targets
#40 

### Description
- Updated Schema
- Modified group_user and group_session migration by adding a default value (which will be equivalent to the current date and time) to the inserted_at and updated_at fields because timestamps() were unable to fill those fields by itself
-  Added validation to some fields
- Modified seeder

### Checklist
- [x] Performed self review of code

### CODE REVIEW

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: lib/dbservice/sessions/session.ex [hunk 1]
```diff
@@ -56,11 +56,23 @@ defmodule Dbservice.Sessions.Session do
...
+    |> validate_date_time
   end
 
   def changeset_update_groups(session, groups) do
     session
     |> change()
     |> put_assoc(:group, groups)
   end
+
+  defp validate_date_time(changeset) do
```
[the remark this answers is not in the document]
Comment (Bahugunajii) [reply lib/dbservice/sessions/session.ex:69]: Done!

File: priv/repo/migrations/20221018090238_modify_group_user.exs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+defmodule Dbservice.Repo.Migrations.ModifyGroupUser do
+  use Ecto.Migration
+
+  def change do
+    alter table(:group_user) do
+      modify :inserted_at, :utc_datetime, default: fragment("NOW()")
+      modify :updated_at, :utc_datetime, default: fragment("NOW()")
```
[the remark this answers is not in the document]
Comment (Bahugunajii) [reply priv/repo/migrations/20221018090238_modify_group_user.exs:7]: Done!

File: lib/dbservice/groups/group.ex [hunk 3]
```diff
@@ -53,5 +52,10 @@ defmodule Dbservice.Groups.Group do
       :group_locale_data
     ])
     |> validate_required([:name, :type])
+    |> validate_program_start_date
+  end
+
+  defp validate_program_start_date(changeset) do
+    invalidate_future_date(changeset, :program_start_date)
```
Comment (rathorevaibhav) [inline lib/dbservice/groups/group.ex:59]: nice!

File: priv/repo/migrations/20221018090242_modify_group_session.exs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+defmodule Dbservice.Repo.Migrations.ModifyGroupSession do
+  use Ecto.Migration
+
+  def change do
+    alter table(:group_session) do
+      modify :inserted_at, :utc_datetime, default: fragment("NOW()")
+      modify :updated_at, :utc_datetime, default: fragment("NOW()")
```
[the remark this answers is not in the document]
Comment (Bahugunajii) [reply priv/repo/migrations/20221018090242_modify_group_session.exs:7]: Yes!

**Review by rathorevaibhav — APPROVED** [review]

**Pull request merged**